package.json

```diff
@@ -90,7 +90,6 @@
     "debug": "2.2.0",
     "minimist": "1.2.0",
     "ramda": "0.21.0",
-    "rollup": "0.37.0",
```

Comment:
@snuggs Easy way - write a documentation about "Rollup must be pre-installed". Harder way - write an issue for NPM and notice them that should be an option in package.json something like "globalDependencies" where all would put dependencies which they rely on.